x/ecocredit/migrations/v3/core_test.go

```diff
@@ -292,3 +318,21 @@ func TestMigrations(t *testing.T) {
 	require.Nil(t, bz)
 
 }
+
+func formatBatchDenom(classId string, batchSeqNo uint64, startDate *time.Time, endDate *time.Time) string {
+	return fmt.Sprintf(
+		"%s-%s-%s-%03d",
+
+		// Class ID string
+		classId,
```

Comment:
```suggestion
func formatBatchDenom(projectId string, batchSeqNo uint64, startDate *time.Time, endDate *time.Time) string {
	return fmt.Sprintf(
		"%s-%s-%s-%03d",

		// Project ID string
		projectId,
```

Reply:
Same here. This is the version we are migrating from.